dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -18,15 +18,18 @@
 
 import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.LinkedList;
```

Comment:
Despite what data structures textbooks say, you almost never need a linked list, and I can't think of any cases where java.util.LinkedList is appropriate. Default to ArrayList.

https://www.youtube.com/watch?v=YQs6IC-vgmo

Reply:
Good to know. I'm picking ArrayDeque then.